### PR TITLE
scrollbar changes

### PR DESCRIPTION
**Pull request by TeerthaDeb**

### CODE REVIEW

**Comment by TeerthaDeb** [comment]
a

File: frontend/package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "zetaforge",
-  "version": "0.3.1",
+  "version": "0.3.2",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "zetaforge",
-      "version": "0.3.1",
```
Comment (FGRCL) [inline frontend/package-lock.json:9]: I think there's an issue with version because npm keeps wanting to upgrade my version as well.

Comment (FGRCL) [reply frontend/package-lock.json:9]: I think you should leave it like that though

**Comment by TeerthaDeb** [comment]
formatted nice and added scrollbar

File: frontend/package-lock.json (hunk 1 above)
Comment (TeerthaDeb) [reply frontend/package-lock.json:9]: okay.

File: frontend/src/styles/globals.scss [hunk 2]
```diff
@@ -131,3 +131,23 @@ main {
...
 }
+
+::-webkit-scrollbar { /* The scrollbar */
+  width: 18px;
+}
+
+::-webkit-scrollbar-track { /* The track */
+  background: var(--cds-background);
+  padding: 1.5px;
+}
```
[the remark this answers is not in the document]
Comment (TeerthaDeb) [reply frontend/src/styles/globals.scss:142]: I would move this and you can see the changes on the next pull.

**Review by FGRCL — APPROVED** [review]
LGTM

**Pull request merged**